website/docs/docs/collaborate/govern/about-model-governance.md

```diff
@@ -12,4 +12,4 @@ pagination_prev: null
 
 [**Model versions**](model-versions): When a breaking change is unavoidable, provide a smoother upgrade pathway by creating a new version of the model. These model versions share a common reference name and can reuse properties & configurations.
 
-[**Project dependencies**](/docs/collaborate/govern/project-dependencies): Use cross project dependencies to reference public models across dbt projects using the [two-argument ref](/reference/dbt-jinja-functions/ref#ref-project-specific-models), which includes the project name.
+[**Project dependencies**](/docs/collaborate/govern/project-dependencies): Use cross project dependencies to reference public models across dbt projects using the [two-argument ref](/reference/dbt-jinja-functions/ref#ref-project-specific-models), which includes the project name. <Lifecycle status='enterprise'/>
```

Comment:
I think the enterprise tag should go after the feature name rather than the paragraph description. Curious how others feel
```suggestion
[**Project dependencies**](/docs/collaborate/govern/project-dependencies)<Lifecycle status='enterprise'/>: Use cross project dependencies to reference public models across dbt projects using the [two-argument ref](/reference/dbt-jinja-functions/ref#ref-project-specific-models), which includes the project name. 
```

Reply:
i kind of prefer it at the end. i've tried having it after the feature and it looked weird. let's give it another try and see what it looks like